src/reduxStore.jsx

```diff
@@ -1,6 +1,6 @@
 import { createStore, applyMiddleware, combineReducers } from 'redux';
 import thunk from 'redux-thunk';
-import reducer from './reducer';
+import transactions from './Stats/reducer';
 
-const mainReducer = combineReducers({ reducer });
+const mainReducer = combineReducers({ transactions });
```

Comment:
please wrap the params in new lines, so later we will see less chnagesing in the pr when we add more reducers.
Each  reducer will go in its own line

Reply:
fixed